### DISCUSSION TITLE
Sind Mitfahrgelegenheiten wie Blablacar und co erwünscht?

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#2791] [Questions / Support]
Es wurden Checkins gemeldet, wo Personen eine Mitfahrgelegenheit im Auto via Blablacar erstellt und darin eingecheckt haben. Den Fall finde ich etwas unschlüssig. Es ist eine Autofahrt (nicht erwünscht), aber durch den Aspekt des "Fremde Person mitnehmen" könnte es etwas aufgeweicht sein? Will man das so scharf abgrenzen? 

[Aktueller Auszug der Regeln](https://help.traewelling.de/rules/#manuelle-fahrten)

(for internal reference: report id 110)

**Comment by pascalwittler** [comment]
Persönlich finde ich die Regeln aktuell relativ eindeutig und deren zugrundeliegendes Prinzip gut, geplante Reisen mit öffentlichen Verkehrsmitteln (im Zweifel also auch Ersatzleistungen in Form von Taxifahrten aus Fahrgastrechte-Ansprüchen) von geplanten, ausdrücklich (und i.d.R. exklusiv nur für sich selbst) gebuchten Autofahrten abzugrenzen.

**Comment by pascalwittler** [comment]
Ein Aufweichen der Regeln für BlaBlaCar würde außerdem Forderungen nach einem Aufweichen für andere Plattformen halbwegs legitim machen, z.B. Uber oder womöglich CarSharing-Anbieter, die z.B. in Karlsruhe auch in den Apps des Verkehrsverbundes aufgeführt werden und von denen man auch behaupten könnte: Ist ja nicht *mein* Auto, sondern da fahren vor und nach mir auch *Fremde* mit.
